src/Form/FormLabel.js

```diff
@@ -8,18 +8,18 @@ import { createStyleSheet } from 'jss-theme-reactor';
 import withStyles from '../styles/withStyles';
 
 export const styleSheet = createStyleSheet('MuiFormLabel', theme => {
-  const focusColor = theme.palette.primary[500];
+  const focusColor = theme.palette.primary[theme.palette.type === 'light' ? 'A700' : 'A200'];
```

Comment:
According to the [latest specs](https://material.io/guidelines/components/text-fields.html#text-fields-states) it should be accent color.
I guess they made that decision to avoid merger with other elements.